
>>> Default:

The error message
Call `rlang::last_error()` to see a backtrace


>>> Reminder:

The error message
Call `rlang::last_error()` to see a backtrace


>>> Reminder (not interactive):

The error message


>>> Branch:

The error message
Backtrace:
 ─with_options(rlang__backtrace_on_error = "branch", msg())
 ─msg()
 ─f()
 ─g()
 ─h()


>>> Collapsed:

The error message
Backtrace:
█
├─[ with_options(...) ]
└─msg()
  ├─[ cat(...) ]
  ├─[ conditionMessage(...) ]
  ├─[ catch_cnd(...) ] with 5 more calls
  └─f()
    ├─[ tryCatch(...) ] with 1 more call
    └─g()
      └─h()
